.github/workflows/binaries.yaml

```diff
@@ -0,0 +1,33 @@
+name: binaries
```

Comment:
maybe better to add to existing https://github.com/opiproject/sztp/blob/main/.github/workflows/release.yaml  file ?

Reply:
I added this to release.yml, is this what you meant?